test/run-tests.c

```diff
@@ -195,5 +195,45 @@ static int maybe_run_test(int argc, char **argv) {
   }
 #endif  /* !_WIN32 */
 
+#if defined(_WIN32) || defined(__linux__)
+  if (strcmp(argv[1], "spawn_helper_affinity") == 0) {
+    int i;
+    int r;
+    int cpu;
+    int cpumask_size;
+#ifdef _WIN32
+    DWORD_PTR procmask;
+    DWORD_PTR sysmask;
+#else
+    cpu_set_t cpuset;
+#endif
+
+    cpumask_size = uv_cpumask_size();
+    ASSERT(cpumask_size > 0);
+
+    cpu = atoi(argv[2]);
+    ASSERT(cpu >= 0);
+    ASSERT(cpu < cpumask_size);
+
+    /* verify the mask has the cpu we expect */
+#ifdef _WIN32
+    r = GetProcessAffinityMask(GetCurrentProcess(), &procmask, &sysmask)?1:0;
```

Comment:
Why the ternary?  Couldn't you just assert that `r != 0`?

Reply:
Oops, that was left from an earlier iteration of the code.  Updated.